core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -552,6 +552,12 @@ public void testToQuery() throws IOException {
         }
     }
 
+    private QueryBuilder<?> rewriteQuery(QB queryBuilder, QueryRewriteContext rewriteContext) throws IOException {
+        QueryBuilder<?> rewritten = QueryBuilder.rewriteQuery(queryBuilder, rewriteContext);
+        assertSerialization(rewritten);
```

Comment:
question: what does it buy us to call `assertSerialization` here if every query has specific serialization tests that already call it? Afraid of forgotten queries, or what else?